src/lua/serializer.h

```diff
@@ -165,6 +167,15 @@ struct luaL_serializer {
 	struct rlist on_update;
 };
 
+/**
+ * @brief helper that sets cfg options default to value.
+ * @param name option to change
+ * @param value new default value
+ * @return non-zero when no option with the name found.
```

Comment:
I think it's better to say that `-1` is returned on error and 0 on success.

Reply:
What is the difference?

Reply:
Changed to `void`.